sites/website/src/docs/community/engineering-process/issue-management.md

```diff
@@ -0,0 +1,120 @@
+---
+id: issue-management
+title: Issue Management
+sidebar_label: Issue Management
+custom_edit_url: https://github.com/microsoft/fast/edit/master/sites/website/src/docs/engineering/issue-management.md
+---
+Managing a popular GitHub repo with a small team is not an easy task. It requires a good balance between creating new features while handling many investigations and bug fixes associated with existing ones.
+
+During the last couple of years the amount of incoming issues has been constantly growing. While this is a sign of a healthy framework and ecosystem surrounding it, it's becoming harder to react to all those issues.
+
+To keep up with ever-evolving expectations, we use a set of rules to help us better handle the incoming issues.
+
+## Goals
+
+The goals of these rules are listed below in priority order:
+
+- Make it easy to make triage decisions for each issue filed in this repository
+- Be able to easily prioritize issues
+- Set proper expectations with customers regarding how issues are going to be handled
+
+## Triage Process Details
+
+The feature teams should be able look through every single issue filed against this repository and be able to make a quick call regarding the nature of the issue.
```

Comment:
Who are the "feature teams"?

Reply:
I'd consider you and Bill a feature team. Nick and Wendy now as well. More broadly my team and Chris team. But I'm trying not to get into all the individual team break down and just generically referring to "feature teams" here.